polaris/evaluate/_metric.py

```diff
@@ -30,6 +31,30 @@ def spearman(y_true: np.ndarray, y_pred: np.ndarray):
     return stats.spearmanr(y_true, y_pred).statistic
 
 
+def absolute_average_fold_error(y_true: np.ndarray, y_pred: np.ndarray):
+    """
+    Calculate the Absolute Average Fold Error (AAFE) metric.
+    It measures the fold change between predicted values and observed values.
+    The implementation is based on https://pubs.acs.org/doi/10.1021/acs.chemrestox.3c00305.
+
+    Args:
+        y_true : array-like of shape (n_samples,)
+            The true target values.
+        y_pred : array-like of shape (n_samples,)
+            The predicted target values.
+
+    Returns:
+        aafe : float
+            The Absolute Average Fold Error.
```

Comment:
We don't annotate the types in the docstring because we use type-hints already!  

```suggestion
    Args:
        y_true: The true target values of shape (n_samples,)
        y_pred: The predicted target values of shape (n_samples,).

    Returns:
        aafe: The Absolute Average Fold Error.
```